e2e/_suites/fleet/features/system_integration.feature

```diff
@@ -0,0 +1,224 @@
+@deploy-system_integration
```

Comment:
To run this feature file in the tests you must add it to the `.ci/.e2e-tests.yml` descriptor, which defines the parallel branches to be spread in the CI workers and OS platforms (Please read https://github.com/elastic/e2e-testing/tree/master/e2e#feature-files-and-the-ci). I'd add this block to the YAML file (from L13 to L34), renaming the main annotation/tag for the feature file:

```yaml
  - suite: "fleet"
    platforms:
      - "arm64"
      - "ubuntu-18.04"
    scenarios:
      - name: "Fleet"
        pullRequestFilter: " && ~debian"
        tags: "fleet_mode_agent"
      - name: "Endpoint Integration"
        platforms:
          - "ubuntu-18.04"
        pullRequestFilter: " && ~debian"
        tags: "agent_endpoint_integration"
      - name: "Linux Integration"
        pullRequestFilter: " && ~debian"
        tags: "linux_integration"
      - name: "System Integration"
        pullRequestFilter: " && ~debian"
        tags: "system_integration"
      - name: "Stand-alone"
        pullRequestFilter: " && ~ubi8"
        tags: "stand_alone_agent"
      - name: "Backend Processes"
        pullRequestFilter: " && ~debian"
        tags: "backend_processes"
```

```suggestion
@system_integration
```

Reply:
Done